### DISCUSSION TITLE
Why is the button element used for language switching in MDN?

### DISCUSSION THREAD

**Discussion by debiru** [mdn/mdn-community#365] [Platform] [answered]
This is one of MDN's pages.

- https://developer.mozilla.org/ja/docs/Web/HTTP/Status/404

There is a language switching list in the upper right corner of the page. However, each language is implemented with a button element, and it is not possible to open pages in a given language in different tabs!

I think this is detrimental to user usability, but why is MDN implemented this way?

If anyone knows anything about this, please let me know.

**Comment by OnkarRuikar** [comment]
Why do you want to open a page in multiple languages at the same time?

**Reply by teoli2003** [reply]
For example, if you want to compare the state of a translation: you want to read the en-US page and, say, the fr page, in parallel.

**Reply by OnkarRuikar** [reply]
If the requirement is coming for translation, then we already have a dev menu dedicated to show the differences:
![app](https://user-images.githubusercontent.com/87750369/232380843-ccc9de6a-e549-4518-8cf9-a73414cabcbc.png)

We can have `View English version` link like we have `View on MDN`.

***

How this is going to be useful for end users(readers)?\
I mean the change is not hard to make, but if we know why we need this then we can provide a better feature.

**Reply by debiru** [reply]
> How this is going to be useful for end users(readers)?

Japanese users often browse the Japanese and English versions in parallel. The current implementation using the button element is inconvenient if you want to display the English version in parallel while viewing the Japanese version. Also, the ability to open in a separate tab is a standard feature supported by browsers. I believe that the unavailability of this function is a degradation of usability.

I have actually seen such voices raised many times, which is why I started this topic.

**Reply by teoli2003** [reply]
I didn't speak about translating the document. As a speaker of several languages, I often look at two different versions of a page (on any website): especially nowadays that companies (not MDN Web docs) use machines for translating content (which often leads to oddities in the translations). That's the same on MDN. If I'm on a French page with something odd, I want to open the English page and compare. Even if I'm not a translator. Now this is painful because it isn't a link.

If semantically it is a link (and it is), let's use a link. It is even easier to style for the devs.

**Comment by debiru** [comment]
Has anyone considered this topic?

I am having trouble opening different language pages in separate tabs when I want to open them.

Why use the button element when you can just use the a (anchor) element?

**Comment by LeoMcA** [comment]
Good question, I'm not sure - I've opened a PR to change them to "proper" links: https://github.com/mdn/yari/pull/9505

**Reply by debiru** [reply]
Thank you for your help. :)

**Comment by LeoMcA** [comment, marked as answer]
PR is now merged and deployed, thanks for pointing this out!

**Reply by debiru** [reply]
Thank you for your follow-up!
